m/mongodb/Dockerfiles/rhel_8.3/docker-entrypoint.sh

```diff
@@ -0,0 +1,383 @@
+#!/bin/bash
+set -Eeuo pipefail
```

Comment:
General comment on this and the dockerfile - has this been adapted from a similar version already available for other architectures elsewhere? If so, we should have a clear attribution as a comment at the top of each file mentioning its origin

Reply:
Add comment with origin information.